security/medusa/l3/comm.c

```diff
@@ -4,12 +4,72 @@
 #include <linux/medusa/l3/arch.h>
 #include <linux/medusa/l3/registry.h>
 #include <linux/medusa/l3/server.h>
+#include <linux/medusa/l4/interface.h>
+
 #include "l3_internals.h"
 
+medusa_answer_t development_converter(authserver_answer_t code)
```

Comment:
add comment to this function

Reply:
I am not sure what kind of comment you have in mind. Comments are useful but also an evil. Those are just helper routines, not core functions. 

If it is not clear from the name what it does I suggest to simply change name to something more specific. The body of the function should not be hard to understand. How does this sound?

Reply:
I mean reasons for conversions... why in development_converter one value is converted do ALLOW, and in production not...